downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -77,6 +77,12 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 ### New features
 
+#### `persist` package
+
+- Introduced support for the h2 data store, mirroring the functionality provided for other supported SQL data stores like MySQL, MSSQL, and PostgreSQL.
+
+### Improvements
```

Comment:
Can you please resolve the conflicts? Looks like this is in the wrong place now too.

Reply:
Fixed in [1703c28](https://github.com/ballerina-platform/ballerina-dev-website/pull/9230/commits/1703c28a3b711cd07ca305eddfee11568f108207)